_posts/2015-07-15-testautomation-allergy.md

```diff
@@ -0,0 +1,60 @@
+
+
+The framework of integrity and constant improvement
+----------------------------------------------------
+
+Why the f*** do we want to repeat a test? Why the f*** do we want to repeat a test? Why the f*** do we want to repeat a test? Why the f*** do we want to repeat a test? 
+
+See how silly it is? If you think of it, there are many variables we don't know. We have randomness in our servers, networks, everywhere! Even bloody electrical current isn't stable. So why on earth do we want to try and repeat the situation over and over, if the world we live in is not repeating itself? You might want to answer "to verify that regression doesn't exist". Isn't the point and purpose of regression testing to see if there is something broken? Shouldn't we be searching for that thing that MIGHT be broken?
+
+Actor Frank Langella said in a New York Times [article](http://www.nytimes.com/2014/01/05/theater/frank-langella-steps-into-king-lear-at-bam.html?hpw&rref=theater&_r=0)
+
+“There’s a tendency to value consistency over creativity. You get it, you nail it, you repeat it. I’d rather hang myself. To me, every night within a certain framework — the framework of integrity — you must forget what you did the night before and create it anew every single time you walk out on the stage.”
+
+This is what test automation does: it finds a way that doesn't break the system and then repeats it. As a tester - as a thinking individual - I choose creativity and ideas before repeatability. I try not to do stupid things, but do the best possible testing every time I do testing. I don't fake it, I try to do the most important thing and I try to communicate my work to other people in a way that makes sense. Test automation doesn't strive for perfection, it settles for acceptable, meager existence.
+
+
+Passion for good testing
+------------------------
+
+"Doing your best work includes having the courage to let go of pretty good work that you’ve done before." - [James Bach](http://www.satisfice.com/blog/archives/1343)
+
+In Solita we take pride in having the courage to do things in ways that are perhaps new and daring. We have passion for what we do. We care about people our work affects. 
+
+I want better automation! I don't want to settle for sub-standard test automation where "every test goes red every time something changes even slightly". Toss that s***! Build robust set of checks and take them for what they are: checks. If you want good testing, have a person do it. The person will most likely see areas that require automation to make testing easier. If you are a programmer who writes test automation, be a professional programmer! Use whatever programming practices you use to make the automation code as good as possible. Start simple, keep it simple. If you don't know what to test or how to test, learn to do testing! I bet there are people willing to help you out if you feel stuck in your quest to test intelligently.
```

Comment:
"Build robust set of checks and take them for what they are: checks" => "Build a robust set of checks and take them for what they are: checks"